feature-libs/product/configurators/variant/src/overview.module.ts

```diff
@@ -15,7 +15,7 @@ import { CmsPageGuard, PageLayoutComponent } from '@spartacus/storefront';
  */
 @NgModule({
   imports: [
-    CommonConfiguratorComponentsModule,
+    CommonConfiguratorModule,
```

Comment:
Do we _really_ need to import this module here?
(same in `interactive-configuration.module.ts`)

Reply:
Yes, this we need. Otherwise the CMS component mapping is not known to the pages.
Probably you were confused because parts of the component mapping is still done here, which is wrong. Nikola suggested that we do this in the component specific modules, but it was not done for all components => will fix it